pandas/tests/extension/test_external_block.py

```diff
@@ -17,9 +17,8 @@ def concat_same_type(self, to_concat, placement=None):
         always 1D in this custom Block
         """
         values = np.concatenate([blk.values for blk in to_concat])
-        return self.make_block_same_class(
-            values, placement=placement or slice(0, len(values), 1)
-        )
+        placement = self.mgr_locs if self.ndim == 2 else slice(len(values))
```

Comment:
you can remove the placement arg from make_block_same_class I think (e.g. do this check internally), but for a followon